### DISCUSSION TITLE
subtitle

### DISCUSSION THREAD

**Discussion by itsmohammadh** [Cyfrin/foundry-full-course-cu#353] [Q&A] [answered]
hi guys, anybody have a this course subtitle? 
my language is not good, so i need the subtitle for understanding course.

**Comment by Naman1729** [comment]
Hey @itsmohammadh , You can Click on the CC (closed captions) 

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/112473210/46e2aa97-298e-404a-8cd4-290b921178c5)

**Reply by itsmohammadh** [reply]
hi @Naman1729 i know that but actually YouTube is filtered in my country and with Vpn YouTube subtitle not working.

**Comment by IAm0xAxe** [comment]
I had the same problem as you, and my solution was to play it using this playlist `https://www.youtube.com/playlist?list=PL2-Nvp2Kn0FPH2xU3IbKrrkae-VVXs1vk` so that the video would be translated with subtitles.

**Reply by itsmohammadh** [reply]
thank you

**Comment by garyghayrat** [comment, marked as answer]
You can download the subtitles here: just click Download Zip and you'll see subtitles for the 3 parts. https://gist.github.com/garyghayrat/5356a71c6eb228c143ae33e8acb72b1b

**Reply by itsmohammadh** [reply]
thanks
